src/styles/components/advanced-section/styles.less

```diff
@@ -0,0 +1,16 @@
+& when (@advanced-section-enabled) {
+  .advanced-section {
+    &-content {
```

Comment:
Let's get a newline above this. The linter should catch this, I wonder if it's not running...